### PR TITLE
Removing Protractor annotations test from e2e, crud, and all test suites due to flakyness

### PR DESCRIPTION
**Pull request by dtaylor113**
We are currently porting over annotation tests to Cypress:  https://issues.redhat.com/browse/CONSOLE-2478

### CODE REVIEW

File: frontend/integration-tests/protractor.conf.ts [hunk 1]
```diff
@@ -59,7 +59,6 @@ const testSuites = {
     'tests/crud.scenario.ts',
     'tests/secrets.scenario.ts',
     'tests/filter.scenario.ts',
-    'tests/modal-annotations.scenario.ts',
```
Comment (spadgett) [inline frontend/integration-tests/protractor.conf.ts:62]: We should leave the scenario and use `xdescribe` so we see they're disabled in the log and JUnit report.

**Review by spadgett — CHANGES_REQUESTED** [review]

File: frontend/integration-tests/protractor.conf.ts (hunk 1 above)
Comment (rhamilto) [reply frontend/integration-tests/protractor.conf.ts:62]: The downside to that is `beforeAll` and `afterAll` will still run...

Comment (rhamilto) [reply frontend/integration-tests/protractor.conf.ts]: > The downside to that is `beforeAll` and `afterAll` will still run...

i should say, `base.scenario.ts` including `Create a test namespace` will still run.

**Review by spadgett — APPROVED** [review]
/lgtm

File: frontend/integration-tests/protractor.conf.ts (hunk 1 above)
Comment (spadgett) [reply frontend/integration-tests/protractor.conf.ts]: I think that only runs once for the entire e2e suite?

Comment (rhamilto) [reply frontend/integration-tests/protractor.conf.ts]: it runs for each suite.  but it's fine.  dave's gonna remove these soon anyhow.

Comment (dtaylor113) [reply frontend/integration-tests/protractor.conf.ts]: yes, and I just checked a few annotation failures, they seem to happen during the actual test, not in before hooks:
```

**Comment by dtaylor113** [comment]
/hold
Until "quay.io has a service degratation incident." is resolved.

**Pull request merged**